src/Card/CardSelect.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Card from 'react-bootstrap/Card';
+import Form from '../Form';
+
+const CardSelect = ({
+  className,
+  cardsData,
+  onChange,
+  selectedCardName,
+}) => {
+  const [currentRecord, setCurrentRecord] = useState(null);
+  useEffect(() => {
+    setCurrentRecord(selectedCardName);
+  }, []);
+
+  return (
+    <>
+      <Form.Group>
+        <Form.RadioSet
+          name="radio-select-card"
+          onChange={(e) => {
+            setCurrentRecord(e.target.value);
+            onChange(e.target.value);
+          }}
+          defaultValue={selectedCardName}
+          isInline
```

Comment:
Radio buttons look sticky with the title. Can we please float them to the left side?

Reply:
Done